src/features/workspaces/components/WorkspaceDrawer.jsx

```diff
@@ -144,21 +165,51 @@ class WorkspaceDrawer extends Component {
             shortcutIndex={0}
           />
           {workspaces.map((workspace, index) => (
-            <WorkspaceDrawerItem
-              key={workspace.id}
-              name={workspace.name}
-              isActive={actualWorkspace === workspace}
-              onClick={() => {
-                if (actualWorkspace === workspace) return;
-                workspaceActions.activate({ workspace });
-                workspaceActions.toggleWorkspaceDrawer();
-              }}
-              onContextMenuEditClick={() =>
-                workspaceActions.edit({ workspace })
-              }
-              services={getServicesForWorkspace(workspace)}
-              shortcutIndex={index + 1}
-            />
+            <div className="wrapper-workspaces-drawer-item">
+              <div className="wrapper-workspaces-drawer-item__workspaces">
+                <WorkspaceDrawerItem
+                  key={workspace.id}
+                  name={workspace.name}
+                  isActive={actualWorkspace === workspace}
+                  onClick={() => {
+                    if (actualWorkspace === workspace) return;
```

Comment:
I would personally place the return within brackets on the next line as this is easier to read.

Reply:
Eslint config can be modified 🌚